clinvar_parser/clinvar-parser/src/main/java/uk/ac/ebi/eva/clinvar/App.java

```diff
@@ -0,0 +1,87 @@
+package uk.ac.ebi.eva.clinvar;
+
+import uk.ac.ebi.eva.clinvar.model.ClinvarSet;
+
+import javax.xml.bind.JAXBException;
+import javax.xml.stream.XMLStreamException;
+import java.io.BufferedWriter;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.zip.GZIPInputStream;
+import java.util.zip.GZIPOutputStream;
+
+public class App {
```

Comment:
Not a big deal but `Application` maybe? They don't charge more for longer words :stuck_out_tongue: